--- a/netstat.c
+++ b/netstat.c
@@ -1742,6 +1742,7 @@
             parsesnmp(flag_raw, flag_tcp, flag_udp);
 #else
             ENOSUPP("netstat", "AF INET");
+            exit(1);
 #endif
         } else if(!strcmp(afname, "inet6")) {
 #if HAVE_AFINET6
@@ -1749,6 +1750,7 @@
             parsesnmp6(flag_raw, flag_tcp, flag_udp);
 #else
             ENOSUPP("netstat", "AF INET6");
+            exit(1);
 #endif
         } else {
           printf(_("netstat: No statistics support for specified address family: %s\n"), afname);
