Undocumented patches found in diff.gz.

--- a/netstat.c
+++ b/netstat.c
@@ -537,7 +537,7 @@
 	printf("%-10lu",inode);
     }
     if (flag_prg)
-	printf(" %-" PROGNAME_WIDTHs "s",prg_cache_get(inode));
+	printf(" %-16s",prg_cache_get(inode));
     if (flag_opt)
 	printf(" %s", timers);
     putchar('\n');
@@ -1815,10 +1815,11 @@
 	    }
 	    printf(_("\nProto Recv-Q Send-Q Local Address           Foreign Address         State      "));	/* xxx */
 	    if (flag_exp > 1)
-		printf(_(" User       Inode     "));
-	    print_progname_banner();
+		printf(_(" User       Inode      "));
+            if (flag_prg)
+                printf(_(" PID/Program name"));
 	    if (flag_opt)
-		printf(_(" Timer"));	/* xxx */
+		printf(_(" Timer"));
 	    printf("\n");
 #else
 	    if (flag_arg) {
